src/main.rs

```diff
@@ -35,12 +35,12 @@ const ERR_BAD_NT: usize = 99;
 /// map a base for indexing the GENETIC_CODE string
 /// x (char): base to look up
 /// returns: usize
-const fn lookup(x: u8) -> usize {
-    match x as char {
-        'T' => 0,
-        'C' => 1,
-        'A' => 2,
-        'G' => 3,
+fn lookup(x: u8) -> usize {
+    match x {
+        65 | 78 => 2,    // A | N 00
+        67 => 1,         // C 01
```

Comment:
You can do `b'C'` instead of 67, easier to read =)

Reply:
Yep, I agree, probably it's better to use `n'C'` for better readability. Thanks @luizirber for the comment !

Reply:
Done. Thank you @luizirber @k3yavi!